### PR TITLE
Setup eventrouter chart

### PR DESCRIPTION
**Pull request by gianrubio**
This is a replacement for #1023

### CODE REVIEW

**Comment by unguiculus** [comment]
Move it to stable directly.

File: stable/eventrouter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels: {{ include "eventrouter.labels" . | indent 4 }}
+  name: {{ template "eventrouter.fullname" . }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "eventrouter.name" . }}
```
Comment (unguiculus) [inline stable/eventrouter/templates/deployment.yaml:10]: Add `release` label to selector.

File: stable/eventrouter/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  template:
+    metadata:
+      labels:
+        app: {{ template "eventrouter.name" . }}
+        release: {{.Release.Name }}
```
Comment (unguiculus) [inline stable/eventrouter/templates/deployment.yaml]: Add space after `{{`.

File: stable/eventrouter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Default values for eventrouter.
+
+image:
+  repository: gcr.io/heptio-images/eventrouter
+  tag: v0.2
+  pullPolicy: Always
```
Comment (unguiculus) [inline stable/eventrouter/values.yaml]: IfNotPresent

File: stable/eventrouter/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    matchLabels:
+      app: {{ template "eventrouter.name" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "eventrouter.name" . }}
+        release: {{.Release.Name }}    
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
```
Comment (unguiculus) [inline stable/eventrouter/templates/deployment.yaml]: Make list indentation consistent throughout the chart. Either use zero or two spaces.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/eventrouter/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for eventruter (https://github.com/heptiolabs/eventrouter)
+name: eventrouter
+version: 0.1.0
+appVersion: 0.2
+home: https://github.com/heptiolabs/eventrouter
+sources:
+- https://github.com/heptiolabs/eventrouter
+keywords:
+- eventrouter
```
Comment (unguiculus) [inline stable/eventrouter/Chart.yaml:10]: Add maintainers

File: stable/eventrouter/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      labels:
+        app: {{ template "eventrouter.name" . }}
+        release: {{ .Release.Name }}    
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
+          - name: config-volume
```
Comment (unguiculus) [inline stable/eventrouter/templates/deployment.yaml]: Indent one more level here for consistency.

File: stable/eventrouter/values.yaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  tag: v0.2
+  pullPolicy: IfNotPresent
+resources: {}
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
```
Comment (unguiculus) [inline stable/eventrouter/values.yaml:14]: Add

```yaml
tolerations: []

nodeSelector: {}
```

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/eventrouter/Chart.yaml (hunk 5 above)
Comment (gianrubio) [reply stable/eventrouter/Chart.yaml:10]: done

File: stable/eventrouter/values.yaml (hunk 7 above)
Comment (gianrubio) [reply stable/eventrouter/values.yaml:14]: done

**Pull request merged**